test/vaadin-list-mixin.html

```diff
@@ -595,5 +616,33 @@
         });
       });
     });
+    describe('vaadin-list-mixin-hidden', () => {
+      let list;
+
+      beforeEach(done => {
+        list = fixture('list-hidden');
+        Polymer.RenderStatus.afterNextRender(list, done);
+      });
+
+      describe('focus', () => {
+
+        beforeEach(() => list._focus(0));
+
+        it('should move focus to next not hidden element on "arrow-down" keydown, from element is not hidden and the next is hidden', () => {
```

Comment:
This description says "keydown" while the other says "keyup", which isn't what's happening here. I think everything after "arrow-down" could be removed. Keep the test names reasonably short.

Reply:
done